linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -68,6 +72,7 @@ impl ClientWrapper {
         network: Network,
         testing_prng_seed: Option<u64>,
         id: usize,
+        close_chains_on_drop: bool,
```

Comment:
nit: We may want to create a (2-value) enum for this.

Reply:
Done.